lib/config/configuration.go

```diff
@@ -427,6 +426,8 @@ func ReadResources(filePath string) ([]types.Resource, error) {
 
 // ApplyFileConfig applies configuration from a YAML file to Teleport
 // runtime config
+//
+// ApplyFileConfig is used by both teleport and tctl binaries.
 func ApplyFileConfig(fc *FileConfig, cfg *servicecfg.Config) error {
```

Comment:
One could argue that tctl using ApplyFileConfig is the real problem, as the function is written mainly with teleport in mind (or at least that was my reading of it).

This is a harder problem to solve (if it's even a problem), so I've opted to move the modules-based check instead.